modular_skyrat/modules/assault_operatives/code/assault_operatives_outfits.dm

```diff
@@ -0,0 +1,136 @@
+//KITS
+/datum/outfit/assaultops
+	name = "I couldn't choose one!"
+
+	mask = /obj/item/clothing/mask/gas/syndicate
+	glasses = /obj/item/clothing/glasses/thermal
+	uniform = /obj/item/clothing/under/syndicate
+	shoes = /obj/item/clothing/shoes/combat
+	gloves =  /obj/item/clothing/gloves/combat
+	back = /obj/item/storage/backpack/fireproof
+	ears = /obj/item/radio/headset/syndicate/alt
+	l_pocket = /obj/item/pinpointer/nuke/goldeneye
+	id = /obj/item/card/id/advanced/chameleon
+	back = /obj/item/mod/control/pre_equipped/nuclear
+	suit_store = /obj/item/gun/ballistic/automatic/pistol/aps
+	r_pocket = /obj/item/ammo_box/magazine/m9mm_aps
+	belt = /obj/item/storage/belt/utility/syndicate
+
+	var/command_radio = FALSE
+	var/cqc = FALSE
+
+/datum/outfit/assaultops/post_equip(mob/living/carbon/human/equipping_human)
+	var/obj/item/radio/radio = equipping_human.ears
+	radio.set_frequency(FREQ_SYNDICATE)
+	radio.freqlock = TRUE
+	if(command_radio)
+		radio.command = TRUE
+
+	if(cqc)
+		var/datum/martial_art/cqc/MA = new
+		MA.teach(equipping_human)
+
+	var/obj/item/implant/weapons_auth/weapons_authorisation = new/obj/item/implant/weapons_auth(equipping_human)
+	weapons_authorisation.implant(equipping_human)
+
+	equipping_human.update_icons()
+
+/datum/outfit/assaultops/cqb
+	name = "Assault Operative - CQB"
+
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/knife/combat,\
+		/obj/item/gun/energy/disabler,\
+		/obj/item/gun/ballistic/automatic/c20r,\
+		/obj/item/ammo_box/magazine/smgm45=4,\
+		)
+
+	cqc = TRUE
+
+/datum/outfit/assaultops/demoman
+	name = "Assault Operative - Demolitions"
+
+	belt = /obj/item/storage/belt/grenade/full
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/knife/combat,\
+		/obj/item/gun/energy/disabler,\
+		/obj/item/gun/ballistic/shotgun/bulldog,\
+		/obj/item/ammo_box/magazine/m12g=4,\
+		/obj/item/implant/explosive/macro, \
+		/obj/item/storage/box/assaultops/demoman
+		)
```

Comment:
Also wouldn't we want the macrobomb implant to be already implanted in them right away? I don't exactly care either way, that being said.
```suggestion
	backpack_contents = list(
		/obj/item/storage/box/survival/syndie,
		/obj/item/knife/combat,
		/obj/item/gun/energy/disabler,
		/obj/item/gun/ballistic/shotgun/bulldog,
		/obj/item/ammo_box/magazine/m12g = 4,
		/obj/item/implant/explosive/macro,
		/obj/item/storage/box/assaultops/demoman,
	)
```